web/html/src/manager/proxy/container-config.tsx

```diff
@@ -0,0 +1,401 @@
+            className="form-control"
+            type="text"
+            placeholder={t("e.g., proxy.admin@mycompany.com")}
+            value={this.state.sslEmail}
+            onChange={this.sslEmailChanged}
+          />
+        </div>
+      </div>,
+    ];
+    var useSSLFields = [
+      <div className="form-group">
+        <label className="col-md-3 control-label">{t("The root CA used to sign the SSL certificate in PEM format")}:</label>
+        <div className="col-md-6">
+          <input
+            name="rootCA"
+            className="form-control"
+            type="file"
+            value={this.state.rootCA}
+            onChange={this.rootCAChanged}
+          />
+        </div>
+      </div>,
+      <div className="form-group">
+        <label className="col-md-3 control-label">{t("intermediate CA used to sign the SSL certificate in PEM format")}:</label>
+        <div className="col-md-6">
+          <input
+            name="intermediateCA"
+            className="form-control"
+            type="file"
+            value={this.state.intermediateCA}
+            onChange={this.intermediateCAChanged}
+          />
+        </div>
+      </div>,
+      <div className="form-group">
+        <label className="col-md-3 control-label">{t("Proxy CRT content in PEM format")}:</label>
+        <div className="col-md-6">
+          <input
+            name="proxyCrt"
+            className="form-control"
+            type="file"
+            value={this.state.proxyCrt}
+            onChange={this.proxyCrtChanged}
+          />
+        </div>
+      </div>,
+      <div className="form-group">
+        <label className="col-md-3 control-label">{t("Proxy SSL private key in PEM format")}:</label>
+        <div className="col-md-6">
+          <input
+            name="proxyKey"
+            className="form-control"
+            type="file"
+            value={this.state.proxyKey}
+            onChange={this.proxyKeyChanged}
+          />
+        </div>
+      </div>,
+    ];
+
+    return (
+      <TopPanel
+        title={t("Container Based Proxy Configuration")}
+        icon="fa fa-cogs"
+        helpUrl="reference/proxy/container-based-config.html"
+      >
+        <p>{t("TODO: some info text message about this page")}</p>
+        {messages}
+        <div className="form-horizontal">
+          <div className="form-group">
+            <label className="col-md-3 control-label">{t("Proxy FQDN")}:</label>
+            <div className="col-md-6">
+              <input
+                name="proxyFQDN"
+                className="form-control"
+                type="text"
+                placeholder={t("e.g., proxy.domain.com")}
+                value={this.state.proxyFQDN}
+                onChange={this.proxyFQDNChanged}
+              />
+            </div>
+          </div>
+          <div className="form-group">
+            <label className="col-md-3 control-label">{t("Max Squid Cache Size (MB)")}:</label>
```

Comment:
Out of curiosity, is there a reason for this to be Papital Case? I don't know the context here, sorry.